src/components/AutoComplete/AutoComplete.test.js

```diff
@@ -0,0 +1,223 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import AutoComplete from './AutoComplete';
+
+test('AutoComplete is working !!', () => {
+  const bsSize = 'large';
```

Comment:
might be better to just inline these in the jsx.  It will put everything in one place, and it is also the closest to how the component will actually be used. As an advantage, you can save some keystrokes on the booleans that are `true` since they can be written like:
```jsx
<AutoComplete allowNew multiple />
```